nntrainer/utils/nntr_threads.cpp

```diff
@@ -13,19 +13,25 @@
 #include <algorithm>
 #include <nntr_threads.h>
 
+#ifdef NNTR_NUM_THREADS
+unsigned int nntr_num_threads = NNTR_NUM_THREADS;
+#else
+unsigned int nntr_num_threads = 1;
+#endif
+
```

Comment:
I'd recomment to constify and make it static with the current usage to avoid future misuse.
```C
static const unsigned int nntr_num_threads = ...
```

Reply:
Fixed. Thanks.